web/app/js/components/util/TapUtils.jsx

```diff
@@ -102,45 +102,45 @@ export const processNeighborData = (source, labels, resourceAgg, resourceType) =
     return resourceAgg;
   }
 
-  let neighb = {};
+  let neighbor = {};
```

Comment:
This abbreviation is odd. A lot of code in this repository and even in this file doesn't use abbreviations. Here, the abbreviation makes it harder to understand the intent.

Is it `neighbors` or `neighbor`? I'm not really sure.